### PR TITLE
Calendar component

### PR DESCRIPTION
**Pull request by AngelaLuo49021**
Screen Shot 2020-02-29 at 1 37 08 PM

Proof it works (console.log was deleted before committng):
Screen Shot 2020-02-29 at 1 41 24 PM

### CODE REVIEW

**Comment by chyku** [comment]
Could you add screen shots + make sure to run formatting?

File: src/components/AssignmentPage/AssignmentPage.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    setDate(value);
+  }
+  return (
+    <div
+      style={{
+        position: "absolute",
+        left: "50%",
+        top: "50%",
+        transform: "translate(-50%, -50%)"
+      }}
```
Comment (chyku) [inline src/components/AssignmentPage/AssignmentPage.js]: why is this inline styled?

File: src/components/CalendarComponent/CalendarComponent.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, { useState, setState, useEffect } from "react";
+import Calendar from "react-calendar/dist/entry.nostyle";
+import "CalendarComponent/CalendarComponent.css";
+import { withFirebase } from "utils/Firebase";
+function CalendarComponent(props) {
```
Comment (chyku) [inline src/components/CalendarComponent/CalendarComponent.js]: hmm could you rename this to Calendar or DateSelector, just following along with Jason's naming standards

File: src/components/AssignmentPage/AssignmentPage.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import CalendarComponent from "../CalendarComponent/CalendarComponent";
+import { Container, Row, Col } from "react-bootstrap";
+import { withFirebase } from "utils/Firebase";
+import { compose } from "recompose";
+import "bootstrap/dist/css/bootstrap.min.css";
+function AssignmentPage() {
```
Comment (chyku) [inline src/components/AssignmentPage/AssignmentPage.js:8]: SICK FUNCTIONAL REACT COMPONENT U GOT THERE

File: src/components/ExampleComponent/ExampleComponent.js [hunk 4]
```diff
@@ -4,14 +4,14 @@ import { withFirebase } from "utils/Firebase";
...
-        return querySnapshot.docs.map(doc =>
-          Object.assign(doc.data(), { id: doc.id })
-        );
-      });
-
+      .then(snapshot =>
+        snapshot.docs.map(d => {
+          const data = d.data();
+          return { id: d.id, ...data };
+        })
```
Comment (chyku) [inline src/components/ExampleComponent/ExampleComponent.js:13]: once the other pr gets merged in this probably wont be a thing or itll cause conflicts just a note

**Review by chyku — CHANGES_REQUESTED** [review]
:fire: great :fire: work :fire:

**Review by rxun — APPROVED** [review]

**Review by chyku — APPROVED** [review]
:fire:

**Pull request merged**